telegram/ext/_commandhandler.py

```diff
@@ -100,16 +108,22 @@ async def callback(update: Update, context: CallbackContext)
         block (:obj:`bool`): Determines whether the return value of the callback should be
             awaited before processing the next handler in
             :meth:`telegram.ext.Application.process_update`.
+        has_args (:obj:`bool` | :obj:`int` | None):
+            Optional argument, otherwise all implementations of :class:`CommandHandler` will break.
+            Defaults to :obj:`None`, which means the handler will process any args or no args.
+
+            .. versionadded:: NEXT.VERSION
     """
 
-    __slots__ = ("commands", "filters")
+    __slots__ = ("commands", "filters", "has_args")
 
     def __init__(
         self,
         command: SCT[str],
         callback: HandlerCallback[Update, CCT, RT],
         filters: Optional[filters_module.BaseFilter] = None,
         block: DVType[bool] = DEFAULT_TRUE,
+        has_args: Union[bool, None, int] = None,
```

Comment:
```suggestion
        has_args: Optional[bool, int] = None,
```

Reply:
Hi, i tried implementing your suggestion, but in return i am failing precommit hook `mypy-ptb` 

<img width="520" alt="image" src="https://github.com/python-telegram-bot/python-telegram-bot/assets/107021083/eb992021-1cd2-44f1-9079-a2d9b559e513">

Suggest to either keep `Union[bool,None,int]` or change to `Optional[Union[bool, int]]` what do you think?

Reply:
oh huh, didn't know that. Ok you can keep as it is then